spec/index.bs

```diff
@@ -7075,27 +7145,36 @@ GPUQueue includes GPUObjectBase;
                 </div>
         </div>
 
-    : <dfn>copyImageBitmapToTexture(source, destination, copySize)</dfn>
+    : <dfn>copyExternalImageToTexture(source, destination, copySize)</dfn>
     ::
-        Schedules a copy operation of the contents of an image bitmap into the destination texture.
+        Issues a copy operation of the contents of a platform image/canvas
+        into the destination texture.
 
-        <div algorithm=GPUQueue.copyImageBitmapToTexture>
+        <div algorithm=GPUQueue.copyExternalImageToTexture>
             **Called on:** {{GPUQueue}} this.
 
             **Arguments:**
-            <pre class=argumentdef for="GPUQueue/copyImageBitmapToTexture(source, destination, copySize)">
-                |source|: {{ImageBitmap}} and origin to copy to |destination|.
+            <pre class=argumentdef for="GPUQueue/copyExternalImageToTexture(source, destination, copySize)">
+                |source|: source image and origin to copy to |destination|.
                 |destination|: The [=texture subresource=] and origin to write to.
                 |copySize|: Extents of the content to write from |source| to |destination|.
             </pre>
 
             **Returns:** {{undefined}}
 
-            If any of the following conditions are unsatisfied, throw an {{OperationError}} and stop.
+            - If |source|.{{GPUImageCopyExternalImage/source}} is not [=origin-compatible=],
+                throw a {{SecurityError}} and stop.
+
+            If any of the following requirements are unmet, throw an {{OperationError}} and stop.
             <div class=validusage>
-                - |copySize|.[=Extent3D/depthOrArrayLayers=] is `1`.
-                - |destination|.{{GPUImageCopyTexture/texture}}.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
-                - |destination|.{{GPUImageCopyTexture/texture}}.{{GPUTexture/[[format]]}} is one of the following:
+                - |copySize|.[=Extent3D/depthOrArrayLayers=] must be `1`.
+                - |destination|.{{GPUImageCopyTexture/texture}}.{{GPUTexture/[[textureUsage]]}}
+                    must include both {{GPUTextureUsage/RENDER_ATTACHMENT}} and
+                    {{GPUTextureUsage/COPY_DST}}.
```

Comment:
Note: We originally added RENDER_ATTACHMENT with video in mind, but I think it's still necessary without, because there is still a possible format conversion in this call (e.g. from rgba8unorm to rg16float, or from rgba16float to rgb10a2unorm).